lib/apiservers/engine/backends/container.go

```diff
@@ -934,347 +967,6 @@ func (c *Container) ContainerAttach(name string, ca *backend.ContainerAttachConf
 	return err
 }
 
-//-------------------------------------
```

Comment:
btw, my goal is to eventually refactor all the personality code and move all remoting code to _proxy.go.  Right now, only remoting code for Create and some of inspect is there.  Once I am done with logs and fix some attach issues, I'm going to go back and go through all of container.go and do the same, and attempt to add unit tests for the other operations.